bc_obps/registration/schema/v2/operation.py

```diff
@@ -39,6 +43,14 @@ class Config:
         ]
         from_attributes = True
 
+    @staticmethod
+    def resolve_sfo_facility_id(obj: Operation) -> Optional[UUID]:
+        # Resolve a single facility id for SFO operations
+        facilities = FacilityDataAccessService.get_current_facilities_by_operation(obj)
+        if obj.type == OperationTypes.SFO.value and len(facilities) > 0:
+            return facilities[0].id
```

Comment:
nit: I suggest using `.count()` instead of `len()` and `.first()` instead of `[0]` when we are dealing with Django QuerySets and we don't want to iterate through it.
[Source](https://docs.djangoproject.com/en/5.1/ref/models/querysets/#:~:text=A%20count()%20call%20performs%20a%20SELECT%20COUNT(*)%20behind%20the%20scenes%2C%20so%20you%20should%20always%20use%20count()%20rather%20than%20loading%20all%20of%20the%20record%20into%20Python%20objects%20and%20calling%20len()%20on%20the%20result%20(unless%20you%20need%20to%20load%20the%20objects%20into%20memory%20anyway%2C%20in%20which%20case%20len()%20will%20be%20faster).)

Reply:
We are pretty sure that we don't pass the `if` statement if `facilities` variable is not truth but seems like mypy is not aware of that :D here is the workaround:
```python
    def resolve_sfo_facility_id(obj: Operation) -> Optional[UUID]:
        # Resolve a single facility id for SFO operations
        facilities = FacilityDataAccessService.get_current_facilities_by_operation(obj)
        if obj.type == OperationTypes.SFO.value and facilities:
            first_facility = facilities.first()
            if first_facility:
                return first_facility.pk
        return None
```

Reply:
Thank you so much, really appreciate it!

Also, I've fixed the cancel button for SFO Facilities and fixed the breadcrumbs 🙇

Reply:
Thanks for tackling all the comments! 🙏